model/material.md

```diff
@@ -0,0 +1,18 @@
+# Material Definition
+
+Simplified compositions:
+
+| Material | Density (g/cc) | Isotopes | Atomic Mass (u)    | Mass Fraction (wt.%) | Atom Density (/barn-cm) |
+|----------|----------------|----------|--------------------|----------------------|-------------------------|
+| UZrH     | 7.2            | 92235    | 235.04393013100002 | 100                  | 1.8447368020026703e-02  |
+| SS304    | 8.0            | 26056    | 55.934936326       | 100                  | 8.61306638827906e-02    |
+| Zr       | 6.5            | 40092    | 91.905034675       | 100                  | 4.343444808841109e-01   |
+| Water    | 1.0            | 1001     | 1.00782503223      | 100                  | 5.975383928175404e-01   |
+| Graphite | 1.7            | 6012     | 12.0               | 100                  | 8.531367125e-02         |
+| B4C      | 2.52           | 5011     | 11.009305355       | 100                  | 1.3784517815293171e-01  |
+| Boral    | 2.53           | 5011     | 37.990843886       | 28.978838659219774   | 4.010444737873965e-02   |
```

Comment:
why the atomic densities of both `B` and `Al` are the same?

Reply:
Because for this pseudo material, number of B atom : number of Al atom = 1:1, which gives those mass fraction.

Reply:
These are calculated by PyNE and I agree them since the atomic ratio is 1:1.
If this is not right, could you show me your values of atom density and the way you calculate them?

Reply:
i see, did know the atomic ratio